lib/views/player_stat_summary.sql

```diff
@@ -19,6 +19,6 @@ SELECT query_username AS username,
 FROM gamer_matches gm
  JOIN matches_augmented m ON gm.match_id = m.match_id
 
-WHERE m.mode NOT LIKE '%plnd%' AND mode NOT LIKE '%jugg%'  AND mode NOT LIKE '%rmbl%'  AND mode NOT LIKE '%mini%' and mode NOT LIKE '%kingslayer%'
+WHERE m.mode NOT LIKE '%plnd%' AND mode NOT LIKE '%jugg%'  AND mode NOT LIKE '%rmbl%'  AND mode NOT LIKE '%mini%' and mode NOT LIKE '%kingslayer%' AND mode NOT LIKE '%dmz%'
```

Comment:
What're your thoughts on sql stored in files like this? I've always thought it was less than ideal, but with Art Lineup, we got kinda crazy with the views and they seemed to get out of hand. Is this more flexible/sustainable than views?

Reply:
This is literally just views. I keep all the files in here incase we need to move databases.